src/prted/pmix/pmix_server_dyn.c

```diff
@@ -533,12 +533,14 @@ static void interim(int sd, short args, void *cbdata)
             if (PMIX_PROC_RANK == info->value.type) {
                 prte_set_attribute(&jdata->attributes, PRTE_JOB_STOP_IN_INIT, PRTE_ATTR_GLOBAL,
                                    &info->value.data.rank, PMIX_PROC_RANK);
+                PMIX_INFO_LOAD(info, PMIX_DEBUG_STOP_IN_INIT, &info->value.data.rank, PMIX_PROC_RANK);
```

Comment:
I'm not sure I understand this - `info` is the incoming object sent to us via `PMIx_Spawn`, so (a) we really don't want to be modifying it, and (b) it already contains the value you are loading here. Can you perhaps explain the reasoning for this change?